udfs/community/month_ceil.sql

```diff
@@ -0,0 +1,25 @@
+/*
+ * Copyright 2019 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+CREATE OR REPLACE FUNCTION fn.month_ceil(dt DATE) RETURNS DATE AS
```

Comment:
I would rename this to LAST_DAY as that's how this function appears in other data warehouses:
* Snowflake - https://docs.snowflake.com/en/sql-reference/functions/last_day.html
* Terdata - https://docs.teradata.com/reader/kmuOwjp1zEYg98JsB8fu_A/FhiYdsm_Lmhy_GaOgXC26A

Also, since this appears in the Teradata migration functions, it would be best to make the Teradata function refer to the community one for consistency.

Reply:
I had no idea this already existed.. and here I was thinking I'd come up with something super unique and novel 😉